fastparquet/util.py

```diff
@@ -120,6 +120,58 @@ def check_column_names(columns, *args):
                                  "" % (missing, arg, columns))
 
 
+def reset_row_idx(data: pd.DataFrame) -> pd.DataFrame:
+    """
+    Shift row index to columns of the DataFrame, compatible for storing to
```

Comment:
I don't understand this description

Reply:
I updated it into:
```python
    Reset row index as a column of the DataFrame.
    Multi-index are stored in columns, one per index level.
```

Reply:
You need to have a single line description and then a blank line